code-challenges.test.js

```diff
@@ -10,20 +10,59 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+/*
+Boiler Plate
+describe("areYouHungry", () => {
+    it("returns eat food or keep coding based on input", () => {
+    expect(areYouHungry("yes")).toEqual("eat food")
+    expect(areYouHungry("no")).toEqual("keep coding")
+    })
+})
+*/
+
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// Expected output example (can be a different order): 
+const colors1Ouptut = ["yellow", "blue", "pink", "green"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// Expected output example (can be a different order): 
+const colors2Output = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+describe("difColor", () => {
+    it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+    expect(difColor(colors1)).toEqual(expect.arrayContaining(colors1Ouptut))
+    expect(difColor(colors2)).toEqual(expect.arrayContaining(colors2Output))
+    })
+})
 
+// ReferenceError: difColor is not defined
 
 // b) Create the function that makes the test pass.
 
+/*
+Create a function named difColor.
+The function that will be created takes in one parameter called 'array'.
+Create a new variable that will take in the length of the initial array and a randomIndex variable
+Using a while loop with the new variable, call the randomIndex variable in order to create a randomize the indexes. Decrement by an individual index
+Return the new array.
+*/
+
+const difColor = (array) => {
+    let arrIndex = array.length, randomIndex;
+    while (arrIndex != 0) {
+        randomIndex = Math.floor(Math.random) * arrIndex
+        arrIndex--
+    }
+    return array
+}
```

Comment:
When I run this code I get back the array in the same exact order. Since the elements contained in the returned array are also the elements contained in the original it passess the test.